server/src/main/kotlin/com/laev/reminder/service/ReviewItemService.kt

```diff
@@ -16,22 +17,24 @@ import com.laev.reminder.repository.ReviewDatetimeRepository
 import com.laev.reminder.service.dto.ReviewItemDetails
 import com.laev.reminder.service.dto.ReviewItemMemorizationCount
 import com.laev.reminder.utils.CycleCalculator
+import com.laev.reminder.utils.DateTimeUtils
 import org.springframework.dao.DataIntegrityViolationException
 import org.springframework.stereotype.Service
 import org.springframework.transaction.annotation.Transactional
 import java.time.OffsetDateTime
 import java.time.ZoneOffset
-import java.time.temporal.ChronoUnit
 
 @Service
 class ReviewItemService(
     private val reviewItemRepository: ReviewItemRepository,
     private val reviewDatetimeRepository: ReviewDatetimeRepository,
     private val memorizationLogRepository: MemorizationLogRepository,
 ) {
+    private val nowDatetime = DateTimeUtils.getCurrentUtcTime()
```

Comment:
This might cause time problem because it's fixed after ReviewItemService instance made.